### PR TITLE
schematrons: Describe the structure and use of phases

### PR DESCRIPTION
**Pull request by bertrand-lorentz**
This is only for 1.10.x, as it describes a change in SDK 1.10.0.

### CODE REVIEW

File: modules/schematrons/pages/index.adoc [hunk 1]
```diff
@@ -24,6 +25,47 @@ The individual Schematron files are executed in a logical sequence:
...
+</phase>
+<phase id="17">
+    <active pattern="EFORMS-validation-stage-1a" />
+    <active pattern="EFORMS-validation-stage-1b-17" />
+    ...
+----
+<1> Definition of a phase with its identifier.
+<2> Pattern containing rules not specific to a subtype.
+<3> Pattern containing rules that apply only to notices with subtype 16.
+
```
Comment (YvesJo) [inline modules/schematrons/pages/index.adoc:59]: Do I understand correctly when I say that:

- the phase ID matches the notice subtype it applies to,
- notice subtype specific set of rules are identifiable with an additional " - _notice-subtype_ " appendix to the pattern?

**Review by YvesJo — APPROVED** [review]
Ok with me

File: modules/schematrons/pages/index.adoc (hunk 1 above)
Comment (bertrand-lorentz) [reply modules/schematrons/pages/index.adoc:59]: Yes, that is correct.
It's just a convention to have unique identifiers. People who use the Schematron don't need to know this, so I haven't documented it.

**Review by pdonohoe — APPROVED** [review]
Nicely described! One minor change of "is" to "are".

**Pull request merged**